airflow/providers/google/cloud/hooks/bigquery.py

```diff
@@ -2133,7 +2140,8 @@ def run_query(
 
         query_param_list: list[tuple[Any, str, str | bool | None | dict, type | tuple[type]]] = [
             (sql, "query", None, (str,)),
-            (priority, "priority", "INTERACTIVE", (str,)),
+            (priority, "priority", priority, (str,)),
+            (location, "location", self.location, (str,)),
```

Comment:
I'm going to revert the location change, in `insert_job` it's not used in `configuration` but rather `jobReference`